lib/project_types/script/script_project.rb

```diff
@@ -30,15 +30,14 @@ def lookup_config(key)
     end
 
     class << self
-      def create(dir)
+      def create(ctx, dir)
         raise Errors::ScriptProjectAlreadyExistsError, dir if Dir.exist?(dir)
-
-        FileUtils.mkdir_p(dir)
-        Dir.chdir(dir)
+        ctx.mkdir_p(dir)
+        ctx.chdir(dir)
       end
 
       def cleanup(ctx:, script_name:, root_dir:)
-        Dir.chdir(root_dir)
+        ctx.chdir(root_dir)
```

Comment:
I think we should update this test [case](https://github.com/Shopify/shopify-app-cli/blob/1d642b4b02d5b8a1b843747b03242acf3fc9a8c9/test/project_types/script/script_project_test.rb#L13).  Also, we should add a test case for `create`.  In general, when you write new code, we should make sure there is proper test coverage in unit tests as much as possible.